config/kwartzlabos.php

```diff
@@ -17,6 +17,9 @@
 
     'entrance_gatekeepers' => [11],             // which gatekeepers are building entrances (used for metrics)
 
+    'membership_coordinator' => [
+        'name' => 'Ian Edington',
```

Comment:
I get why this is here but this is really application data that (with a lot more work) should probably be managed by the application instead of as source code. This title is dynamic but slow enough to change that people will likely make the edit then forget where the setting is set by the time the term changes again.

Reply:
I agree it should be application data and I don't want to do that work now. I do think it will be easy enough to do a `git grep 'Ian Edington'` and find this instance.